src/test/java/com/znsio/teswiz/services/UnirestService.java

```diff
@@ -0,0 +1,38 @@
+package com.znsio.teswiz.services;
+
+import kong.unirest.HttpResponse;
+import kong.unirest.JsonNode;
+import kong.unirest.Unirest;
+import org.apache.log4j.Logger;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class UnirestService {
+
+    private static final Logger LOGGER = Logger.getLogger(UnirestService.class);
+    private static final Unirest unirest = new Unirest();
+
+    private static Unirest getUnirestObj(){
```

Comment:
need to handle proxy situation

Reply:
For current API scenarios and workflows it is working without adding proxy configurations.

Reply:
For Proxy, which proxy to use? also, we do not have any api currently to test with proxy. could you suggest something around this. also for open source APIs we are using , proxy is not needed. so please suggest.